test/framework/cloudstack.go

```diff
@@ -365,6 +375,11 @@ func (c *CloudStack) Redhat129Template() api.CloudStackFiller {
 	return c.templateForKubeVersionAndOS(anywherev1.Kube129, RedHat9, nil)
```

Comment:
Looks like I made a mistake here when adding the 1.29 E2E tests but apparently this was being used only in one spot `TestCloudStackKubernetes128To129Redhat8UnstackedEtcdUpgrade` and that one was passing too. But let's update it to the correct one.
```suggestion
	return c.templateForKubeVersionAndOS(anywherev1.Kube129, RedHat8, nil)
```